tests/integration/components/ice-popover-test.js

```diff
@@ -315,3 +315,36 @@ test('First item autofocuses when opened by keyboard only', async function(asser
   assert.ok(popover.isOpen, 'popover rendered on click');
   assert.ok(!popover.content.buttonWithFocus.isPresent, 'first button does not have focus');
 });
+
+moduleForComponent('ice-popover', 'Unit | Component | ice-popover', {
+  unit: true
+});
+
+test('popper modifiers are customizable', function(assert) {
+  assert.expect(2);
+
+  let popover = this.subject({});
```

Comment:
Let's avoid doing `this.subject` here. We can turn this into a normal unit test.